packages/suite/src/views/dashboard/components/NewsFeed/index.tsx

```diff
@@ -81,11 +83,14 @@ interface Props {
     maxVisibleCount?: number;
 }
 
+const BLOG_URL = 'https://blog.trezor.io/';
```

Comment:
Already defined here: https://github.com/trezor/trezor-suite/blob/d3a5d232f105d6ca53e15dbed52848e677eb3771/packages/suite/src/constants/suite/urls.ts#L8

Reply:
Addressed in latest rebase/squash (44cf70a)